docs/tutorials/tutorial-sql-null.md

```diff
@@ -0,0 +1,214 @@
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+This tutorial introduces the basic concepts of null handling for string and numeric columns in Apache Druid.
+The tutorial focuses on filters using the logical NOT operation on columns with NULL values.
+
+## Prerequisites
+
+Before starting this tutorial, download and run Apache Druid on your local machine as described in
+the [Local quickstart](index.md).
+
+The tutorial assumes you are familiar with using the [Query view](./tutorial-sql-query-view.md) to ingest and query data.
+
+The tutorial also assumes you have not changed any of the default settings for null handling.
+
+## Load data with null values
+
+The tutorial loads some data with null values for string and numeric columns as follows:
+
+```json
+{"date": "1/1/2024 1:02:00","title": "example_1","string_value": "some_value","numeric_value": 1}
+{"date": "1/1/2024 1:03:00","title": "example_2","string_value": "another_value","numeric_value": 2}
+{"date": "1/1/2024 1:04:00","title": "example_3","string_value": "", "numeric_value": null}
+{"date": "1/1/2024 1:05:00","title": "example_4","string_value": null, "numeric_value": null}
+```
+
+Run the following query in the Druid Console to load the data:
+
+```sql
+REPLACE INTO "null_example" OVERWRITE ALL
+WITH "ext" AS (
+  SELECT *
+  FROM TABLE(
+    EXTERN(
+      '{"type":"inline","data":"{\"date\": \"1/1/2024 1:02:00\",\"title\": \"example_1\",\"string_value\": \"some_value\",\"numeric_value\": 1}\n{\"date\": \"1/1/2024 1:03:00\",\"title\": \"example_2\",\"string_value\": \"another_value\",\"numeric_value\": 2}\n{\"date\": \"1/1/2024 1:04:00\",\"title\": \"example_3\",\"string_value\": \"\", \"numeric_value\": null}\n{\"date\": \"1/1/2024 1:05:00\",\"title\": \"example_4\",\"string_value\": null, \"numeric_value\": null}"}',
+      '{"type":"json"}'
+    )
+  ) EXTEND ("date" VARCHAR, "title" VARCHAR, "string_value" VARCHAR, "numeric_value" BIGINT)
+)
+SELECT
+  TIME_PARSE("date", 'd/M/yyyy H:mm:ss') AS "__time",
+  "title",
+  "string_value",
+  "numeric_value"
+FROM "ext"
+PARTITIONED BY DAY
+```
+
+After Druid finishes loading the data, run the following query to see the table:
+
+```sql
+SELECT * FROM "null_example"
+```
+
+|`__time`|`title`|`string_value`|`numeric_value`|
```

Comment:
The table has empty columns: 
![image](https://github.com/apache/druid/assets/53799971/6270665e-bb97-43a8-89fd-c984f7a69035)